includes/admin/customers/customers.php

```diff
@@ -773,23 +779,25 @@ function edd_customers_view( $customer = null ) {
 
 					<tr data-key="<?php echo esc_attr( $key ); ?>">
 						<td class="column-actions">
-							<span><?php echo esc_html( $email ); ?></span>
+							<span><?php echo esc_html( $email->email ); ?></span>
 
-							<?php if ( 'primary' === $key ) : ?>
+							<?php if ( 'primary' === $email->type ) : ?>
 								<span class="edd-chip"><?php esc_html_e( 'Primary', 'easy-digital-downloads' ); ?></span>
+							<?php else : ?>
+								<div class="row-actions">
+									<?php
+									$base_url    = admin_url( 'edit.php?post_type=download&page=edd-customers&view=overview&id=' . urlencode( $customer->id ) );
```

Comment:
Not a big deal either way, but could this use `add_query_arg` instead? I find it easier to parse mentally, or actually maybe just use the `edd_get_admin_url` function to build this base URL.